src/zenml/cli/stack_recipes.py

```diff
@@ -0,0 +1,1013 @@
+            self.stack_recipe_repo.clone()
+
+        try:
+            self.stack_recipe_repo.checkout(branch=branch)
+        except GitCommandError:
+            cli_utils.warning(
+                f"The specified branch {branch} not found in "
+                "repo, falling back to the latest release."
+            )
+            self.stack_recipe_repo.checkout_latest_release()
+
+    def pull_latest_stack_recipes(self) -> None:
+        """Pulls the latest stack recipes from the stack recipes repository."""
+        self.pull(
+            branch=self.stack_recipe_repo.latest_release_branch, force=True
+        )
+
+    def copy_stack_recipe(
+        self, stack_recipe: StackRecipe, destination_dir: str
+    ) -> None:
+        """Copies an stack recipe to the destination_dir.
+
+        Args:
+            stack_recipe: The stack recipe to copy.
+            destination_dir: The destination directory to copy the recipe to.
+        """
+        io_utils.create_dir_if_not_exists(destination_dir)
+        io_utils.copy_dir(
+            str(stack_recipe.path_in_repo), destination_dir, overwrite=True
+        )
+
+    def clean_current_stack_recipes(self) -> None:
+        """Deletes the ZenML stack recipes directory from your current working directory."""
+        stack_recipes_directory = os.path.join(
+            os.getcwd(), "zenml_stack_recipes"
+        )
+        shutil.rmtree(stack_recipes_directory)
+
+
+pass_git_stack_recipes_handler = click.make_pass_decorator(
+    GitStackRecipesHandler, ensure=True
+)
+pass_tf_client = click.make_pass_decorator(Terraform, ensure=True)
+
+
+@stack.group("recipe", cls=TagGroup)
+def stack_recipe() -> None:
+    """Access all ZenML stack recipes."""
+
+
+@stack_recipe.command(name="list", help="List the available stack recipes.")
+@pass_git_stack_recipes_handler
+def list_stack_recipes(
+    git_stack_recipes_handler: GitStackRecipesHandler,
+) -> None:
+    """List all available stack recipes.
+
+    Args:
+        git_stack_recipes_handler: The GitStackRecipesHandler instance.
+    """
+    cli_utils.warning(ALPHA_MESSAGE)
+    stack_recipes = [
+        {"stack_recipe_name": stack_recipe.name}
+        for stack_recipe in git_stack_recipes_handler.get_stack_recipes()
+    ]
+    cli_utils.print_table(stack_recipes)
+
+    cli_utils.declare("\n" + "To get the latest list of stack recipes, run: ")
+    text = Text("zenml stack recipe pull -y", style="markdown.code_block")
+    cli_utils.declare(text)
+
+    cli_utils.declare("\n" + "To pull any individual stack recipe, type: ")
+    text = Text(
+        "zenml stack recipe pull RECIPE_NAME", style="markdown.code_block"
+    )
+    cli_utils.declare(text)
+
+
+@click.option(
+    "--path",
+    "-p",
+    type=click.STRING,
+    default="zenml_stack_recipes",
+    help="Relative path at which you want to clean the stack_recipe(s)",
+)
+@stack_recipe.command(help="Deletes the ZenML stack recipes directory.")
```

Comment:
Apparently this works as is, but I think it would be nicer for readability to have the `.command()` above all the possible options

Reply:
agreed!